<?xml version="1.0" ?><root>
<report><version>2</version><sessionid>81d9aa0dc4c0abb0664d10be</sessionid><type>desync</type><sku>ea.maxis.sims4.15.macx86</sku><createtime>2024-05-22 10:53:30</createtime><buildsignature>Local.Unknown.Unknown.1.106.148.1230-1.200.000.623.Release</buildsignature><categoryid>__init__.py:202</categoryid><desyncid>81d9aa0dc4c0abb0664d10be</desyncid><systemconfig/><screenshot/><desyncdata>Exception thrown while starting situation (AttributeError: 'object_Baby' object has no attribute 'is_being_destroyed')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 130, in wrapped&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 126, in finally_wrap&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 157, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 175, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 383, in c_api_client_connect&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\zone.py", line 609, in do_zone_spin_up&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\zone_spin_up_service.py", line 1654, in update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\zone_spin_up_service.py", line 969, in on_enter&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\game_object.py", line 1438, in finalize&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\script_object.py", line 2397, in finalize&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\baby\baby.py", line 597, in on_finalize_load&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\baby\baby.py", line 332, in set_sim_info&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\baby\baby.py", line 500, in enable_baby_state&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\buff_component.py", line 229, in on_bassinet_ready_to_simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\buffs\buff.py", line 1476, in on_bassinet_ready_to_simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\buffs\buff.py", line 1744, in _start_auto_situation&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\situations\tunable.py", line 277, in &lt;lambda&gt;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\situations\tunable.py", line 254, in start_situation&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\situations\situation_manager.py", line 666, in create_situation&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\situations\situation_manager.py", line 815, in create_situation_from_seed&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\situations\complex\caregiver_situation.py", line 475, in start_situation&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\situations\situation_complex.py", line 1146, in _change_state&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\situations\complex\caregiver_situation.py", line 40, in on_activate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\situations\complex\caregiver_situation.py", line 274, in on_enter_passive_state&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\situations\complex\caregiver_situation.py", line 352, in _update_caregiver_status&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\situations\complex\caregiver_situation.py", line 255, in get_care_dependent_sim_info&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\__init__.py", line 202, in __getattr__&#13;&#10;AttributeError: 'object_Baby' object has no attribute 'is_being_destroyed'rtim=0&#13;&#10;Client Session Time: 2091.55&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xbe7da869&#13; &#10;Game Sub-state: 0x1020b807&#13; &#10;LoadingScreenActiveFlag: 0&#13; &#10;Origin Version: 13,194,0,5707&#13; &#10;Modded: False&#13; &#10;SystemInfo: Mac OS X Version 11.7.4 (Build 20G1120) GPU: Intel UHD Graphics 617 macOS Metal driver 0.0.0.0&#13; &#10;Architecture: x86_64&#13; &#10;</desyncdata></report>
</root>